Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/special.yml

```diff
@@ -48,10 +48,26 @@
 - type: entity
   id: BulletSpecialMindbreaker
   name: bullet (.38 special mindbreaker)
-  parent: BaseBullet
+  parent: BaseBulletPractice
   noSpawn: true
   components:
   - type: Projectile
     damage:
       types:
-        Blunt: 3
+        Blunt: 1
+  - type: SolutionContainerManager
```

Comment:
Actually come to think of it, this doesn't contain any chemicals either. To fix that you'll need to add a reagents field to SolutionContainerManager. Since it would then come preloaded with the apropriate chems, you won't need RefillableSolution or DrainableSolution either. 

![image](https://github.com/DeltaV-Station/Delta-v/assets/16548818/1b29620c-05a9-4c84-9b6d-ed45b3c305c4)